frontend/src/app/main/ui/comments.scss

```diff
@@ -112,150 +106,147 @@
 }
 
 .replies {
+  @include bodySmallTypography;
   display: flex;
   gap: $s-8;
 }
 
-.total-replies {
+.replies-total {
   color: var(--color-foreground-secondary);
 }
-.new-replies {
+.replies-unread {
   color: var(--color-accent-primary);
 }
-// Thread-bubble
 
-.floating-thread-bubble {
-  @extend .comment-bubbles;
+.floating-preview-wrapper {
+  z-index: 1;
   position: absolute;
+  user-select: none;
   cursor: pointer;
   pointer-events: auto;
   transform: translate(calc(-1 * $s-16), calc(-1 * $s-16));
+}
 
-  &.resolved {
-    @extend .resolved-comment-bubble;
-  }
-  &.unread {
-    @extend .unread-comment-bubble;
-  }
+.floating-preview-bubble {
+  z-index: 0;
```

Comment:
Maybe it's better to put it as `initial` instead of the hard-coded value `0`. Another option is to change it to a variable.

Reply:
Fixed